paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -133,6 +133,14 @@
     data_type : dtype
     backend : place > output
 
+- op : batch_fc
+  args : (Tensor input, Tensor w, Tensor bias)
+  output : Tensor(out)
+  infer_meta:
+    func : BatchFCInferMeta
+  kernel :
+    func : batch_fc
```

这里需要添加下kernel的dtype
```suggestion
    func : batch_fc
    data_type: input
```
因为它的原OP定义里包含GetKernelType，该字段用于选择kernel，表明kernel_dtype会和input.dtype保持一致

---

其他地方没啥问题，把这里改下就好了